docker-compose.yml

```diff
@@ -1,60 +1,81 @@
 # Note: Requires docker 1.10.0+ and docker-compose 1.6.0+.
-version: "2"
+version: '3.4'
 services:
   db:
     image: postgres:9.4
 
-  # run the site with the django dev-server
-  web:
-    image: local/standup_dev
+  # Run browser-sync and compile static assets.
+  #
+  # This is the primary server and it proxies
+  # the Django app so that it can add the fancy
+  # browser refresh JS.
+  assets:
+    build:
+      context: .
+      target: assets
+    image: mozmeao/standup_assets:latest
     ports:
-      - "8000:8000"
+      - "3000:3000"
+      - "3001:3001"
     volumes:
-      - .:/app
-    links:
-      - db
+      - ./static/:/app/static:cached
+      - ./standup/:/app/standup:cached
+    environment:
+      BS_PROXY_URL: "app:8000"
+      BS_OPEN_BROWSER: "false"
+    command: npm run docker-start
+
+  # the django app
+  app:
+    build:
+      context: .
+      target: devapp
+    image: mozmeao/standup_dev:latest
+    command: ./bin/run.sh dev
+    env_file: .env
     environment:
       - DATABASE_URL=postgres://postgres@db/postgres
       - DEBUG=True
-      - ALLOWED_HOSTS=localhost,127.0.0.1,web
+      - ALLOWED_HOSTS=localhost,127.0.0.1,app
       - SECRET_KEY=itsasekrit
-    command: ./bin/run.sh dev
+      - USER=${USERID:-1000}
+    depends_on:
+      - db
+    ports:
+      - "8000:8000"
+    volumes:
+      - .:/app
 
   # run the site in a prod-like environment
   prod:
-    image: local/standup_base
+    build:
+      context: .
+      target: webapp
+    image: mozmeao/standup:latest
     ports:
       - "8000:8000"
-    # volumes:
-    #   - .:/app
+    volumes:
+      - .:/app
     links:
       - db
     environment:
       - DATABASE_URL=postgres://postgres@db/postgres
       - DEBUG=False
       - ALLOWED_HOSTS=localhost,127.0.0.1
       - SECRET_KEY=itsasekrit
+      - USER=${USERID:-1000}
```

Comment:
`USERID` is now a uid:gid combo, so this won't work.

I did the uid:gid combo in antenna, because I was using `-u` which takes a uid:gid combo. If you want to thread things through using this `USER` environment variable (which is cool), then you want to change the `USERID` to just be the uid.

Reply:
Oh right. I was thinking it would work fine because `gosu` supports that, but I'm not using it that way :). I'll fix.

Reply:
Also I would just use `-u` but that seems to also break things since that user doesn't exist in the container. Not sure if we could change something more simple to make that work, but this does seem to work to create the user on the fly.